exercise12_5.py

```diff
@@ -6,24 +6,23 @@
 Python for Everybody: Exploring Data Using Python 3
 by Charles R. Severance
 
-Solution by Jamison Lahman, June 5, 2017
+Solution by Daniel Lee, Dec 29, 2019
```

Comment:
Prefer not to change this. Github revision history should be sufficient. If you'd really link,
```
Solution by Jamison Lahman, June 5, 2017
Additional contribution by Daniel Lee, Dec 29, 2019
```